app/views/drawings/_drawings.html.haml

```diff
@@ -1 +1,51 @@
-= render @drawings.desc
+.container
+  %h3 Manage Drawings
+  %table.table.table-bordered.table-condensed.table-hover.table-responsive
+    %thead
+      %tr
+        %th ID
+        %th File
+        %th Subject Matter
```

Comment:
OH! I thought it was some sort of extended description of the drawing. Really didn't realise it was referring to the artist 🙈. Probably should've read the `placeholder` in the upload view.

Hmmm... how about `Artist's Background`?